docs/Secure-Coding-Guide-for-Python/CWE-703/CWE-754/noncompliant01.py

```diff
@@ -1,19 +1,35 @@
 # SPDX-FileCopyrightText: OpenSSF project contributors
 # SPDX-License-Identifier: MIT
-""" Non-compliant Code Example """
+"""Non-compliant Code Example"""
+
 import sys
 
 
 class Package:
+    """Class representing a package object"""
+
     def __init__(self):
-        self.package_weight = float(1.0)
+        self.package_weight = float(0.0)
+        self.max_package_weight = 100.0
 
-    def put_in_the_package(self, object_weight):
+    def add_to_package(self, object_weight: str):
+        """Function for adding an object into the package"""
         value = float(object_weight)
-        print(f"Adding an object that weighs {value} units")
+        # This is dead code as value gets type cast to float,
+        # hence will never be equal to string "NaN"
+        if value == "NaN":
+            raise ValueError("'NaN' not a number")
+        # This is also dead code as value is type cast to float,
+        # unusual inputs like -infinity will not get caught
+        if isinstance(value, float) is not True:
```

Comment:
Same suggestion as in README:
```suggestion
        if isinstance(value, float) is False:
```